angular-ngc/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_config")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@aspect_rules_js//npm:defs.bzl", "npm_link_package")
+
+# Link npm packages
+npm_link_all_packages(name = "node_modules")
+
+# The root repo tsconfig
+ts_config(
+    name = "tsconfig",
+    src = "tsconfig.json",
+    visibility = ["//visibility:public"],
+    deps = ["tsconfig.base.json"],
+)
+
+ts_config(
+    name = "tsconfig.node",
+    src = "tsconfig.node.json",
+    visibility = ["//visibility:public"],
+    deps = ["tsconfig.base.json"],
+)
+
+# Repo NPM packages. Assumes a common bazel rule format
+# of "//{path}:npm-pkg" based on ng_pkg().
+PACKAGES = {
+    "@ngc-example/common": "packages/common",
+    "@ngc-example/lib-a": "packages/lib-a",
+}
+
+[
+    npm_link_package(
```

Comment:
Did you try using pnpm workspaces for the 1pm deps? The linking would happen automatically in `npm_link_all_packages` and it would make editor support work for the `1p` deps when you run `pnpm install`

Reply:
I can help with switching that in a follow-up if you want to get this landed today

Reply:
Is that the preferred method? That seems to require repeating the package name in even more places?

Reply:
Its the happy path and it will make editor play nice with 1p deps

Reply:
Should end up with less boilerplate